references/classification/train.py

```diff
@@ -127,7 +127,7 @@ def load_data(traindir, valdir, args):
     if args.cache_dataset and os.path.exists(cache_path):
         # Attention, as the transforms are also cached!
         print(f"Loading dataset_train from {cache_path}")
-        dataset, _ = torch.load(cache_path)
+        dataset, _ = torch.load(cache_path, weights_only=True)
```

Comment:
Should we check if ref script is still working with this update?

Reply:
Yes, certainly. Caching the dataset is not that common I guess and there were some discussion regarding removing it (https://github.com/pytorch/vision/issues/6727#issuecomment-1272945951). However, I agree that this is likely something that could break with `weights_only`.

Reply:
We can change to `weights_only=False` if it's required, just be explicit about it.